src/main/cluster.ts

```diff
@@ -14,13 +14,27 @@ import { getFeatures, installFeature, uninstallFeature, upgradeFeature } from ".
 import request, { RequestPromiseOptions } from "request-promise-native"
 import { apiResources } from "../common/rbac";
 import logger from "./logger"
+import { VersionDetector } from "./detectors/version-detector";
```

Comment:
"detectors" sounds a bit too generic name for these... maybe "cluster-detectors"?

Reply:
renamed